src/main/java/org/influxdb/dto/QueryResult.java

```diff
@@ -3,6 +3,8 @@
 import java.util.List;
 import java.util.Map;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
Why do we introduce a new dependency to jackson ?

Reply:
because the converter https://github.com/komamitsu/retrofit-converter-msgpack (you proppsed it) depends on jackson

Reply:
OK, i didnt check that, my fault. But still, using another json converter with this change is not desired. We must think twice "